36/documentation.html

```diff
@@ -54,12 +54,10 @@ <h3>Kafka 3.6 Documentation</h3>
                     <a href="/26/documentation.html">2.6.X</a>, 
                     <a href="/27/documentation.html">2.7.X</a>,
                     <a href="/28/documentation.html">2.8.X</a>,
-                    <a href="/30/documentation.html">3.0.X</a>,
-                    <a href="/31/documentation.html">3.1.X</a>,
-                    <a href="/32/documentation.html">3.2.X</a>,
-                    <a href="/33/documentation.html">3.3.X</a>,
-                    <a href="/34/documentation.html">3.4.X</a>,
-                    <a href="/35/documentation.html">3.5.X</a>.
+                    <a href="/30/documentation.html">3.0.X</a>.
+                    <a href="/31/documentation.html">3.1.X</a>.
+                    <a href="/32/documentation.html">3.2.X</a>.
+                    <a href="/33/documentation.html">3.3.X</a>.
```

Comment:
This is the 3.6 docs, so it should point to all previous releases including 3.4 and 3.5. Why are we removing them?

Reply:
yes, I don't intend to - I captured it in https://github.com/apache/kafka-site/pull/586/files#r1499488695 - that commit is missing.